pcdsdevices/beam_stats.py

```diff
@@ -9,6 +9,9 @@ class BeamStats(Device):
     ev = Cpt(EpicsSignalRO, 'SIOC:SYS0:ML00:AO541')
     rate = Cpt(EpicsSignalRO, 'EVNT:SYS0:1:LCLSBEAMRATE')
     owner = Cpt(EpicsSignalRO, 'ECS:SYS0:0:BEAM_OWNER_ID')
+    # This one is much faster than ev
+    accel_ev = Cpt(EpicsSignalRO, 'BLD:SYS0:500:PHOTONENERGY')
+    sxr_gmd = Cpt(EpicsSignalRO, 'SXR:GMD:BLD:milliJoulesPerPulse')
```

Comment:
This is good to have. In the future we might want a full `SxrGmd` Device with all of the sxr statistics. I'll leave it up to you if you want to start that now or wait to see if it's needed.

I'm bringing this up because `sxr_gmd` seems like the name of the full device, rather than just the mJ measurement.